test/libsolidity/semanticTests/array/copying/copy_byte_array_in_struct_to_storage.sol

```diff
@@ -37,11 +37,11 @@ contract C {
 // compileViaYul: also
 // ----
 // f() -> 0x40, 0x80, 6, 0x6162636465660000000000000000000000000000000000000000000000000000, 0x49, 0x3132333435363738393031323334353637383930313233343536373839303120, 0x3132333435363738393031323334353637383930313233343536373839303120, 0x3132333435363738390000000000000000000000000000000000000000000000
-// gas irOptimized: 179963
```

Comment:
These gas savings come from the fact that the load resolver can now reason that two memory relations are unrelated:
```
                let zero_struct_S_mpos := allocate_and_zero_memory_struct_struct_S()
                mstore(zero_struct_S_mpos,  0x07)
                let _1 := add(zero_struct_S_mpos, 96)
                mstore(_1, copy_literal_to_memory_e3fc88296de5f3bc29e06c9130a9b57250926f606654c95ecdac75bb501a42ff())
                let _2 := 32
                let _3 := add(zero_struct_S_mpos, _2)
                let memPtr := allocate_memory_array_string_1990()
                mstore(add(memPtr,  _2),  "abcdef")
                mstore(_3, memPtr)
                let _4 := add(zero_struct_S_mpos, 64)
                mstore(_4,  0x09)
                update_storage_value_offsett_uint16_to_uint16_1991(and(mload(zero_struct_S_mpos), 0xffff))
                let _5 := mload(_3) /* XXX */
                let newLen := mload(_5)
```

This is the old code. In the new code, the load marked by `XXX` is replaed by `memPtr`.